tf-psa-crypto/drivers/builtin/src/aesni.c

```diff
@@ -489,7 +489,7 @@ int mbedtls_aesni_crypt_ecb(mbedtls_aes_context *ctx,
          "movdqu    %%xmm0, (%4)    \n\t" // export output
          :
```

Comment:
Please [add a changelog entry file](https://github.com/Mbed-TLS/mbedtls/blob/development/ChangeLog.d/00README.md). Even if we don't know for sure that a platform is affected, insufficient clobbers are a bug. If the next GCC/Clang/MSVC/… triggers the bug, users should be informed of which version of Mbed TLS fixed it.

Reply:
Under Bugfix, please.

Looking at the history, it seems we messed up the changelog sections in the 2.17.0 release. [Originally](d20a0e2d9f371ed20d6379d50654724580c58655) that entry was under Bugfix.